dbux-data/src/DataProviderBase.js

```diff
@@ -296,13 +304,13 @@ export default class DataProviderBase {
         ];
       }))
     };
-    return JSON.stringify(obj);
+    return obj;
   }
 
   /**
-   * Use: `dataProvider.deserialize(JSON.parse(stringFromFile))`
+   * Use: `dataProvider.deserializeJson(JSON.parse(serializedString))`
    */
-  deserialize(data, applyPostAdd = false) {
+  deserializeJson(data) {
```

Comment:
I really like the new naming scheme (`raw` vs `processed`)! It already automatically told us that `raw` will always be `false` in this case anyway :)

(This is not a "review comment", just a normal comment; but Github apparently does not allow "non-review comments", once a review has started :/)